roles/kubernetes-apps/external_cloud_controller/huaweicloud/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+reviewers:
+  - alijahnas
+  - luckySB
```

Comment:
Not sure if those are really relevant :thinking: (I am not the best person to know that though :see_no_evil:)

Reply:
OK me neither. 🤷  I'm open for suggestions ;)

Reply:
What are you adding specific OWNERS for this ?

Reply:
I'm unsure of the need so I copied this one from the OpenStack provider. If it's not necessary I'll delete it!